runtime/oti/jvmtiInternal.h

```diff
@@ -516,7 +516,7 @@ typedef struct jvmtiGcp_translation {
 #define ENSURE_VALID_HEAP_OBJECT_FILTER(var)
 #define ENSURE_MONITOR_NON_NULL(var)
 #if JAVA_SPEC_VERSION >= 19
-#define ENSURE_JTHREAD_NOT_VIRTUAL(vmThread, jthrd)
+#define ENSURE_JTHREAD_NOT_VIRTUAL(vmThread, jthrd, error)
 #define ENSURE_JTHREADOBJECT_NOT_VIRTUAL(vmThread, jthrdObject)
```

Comment:
I suggest the same change should be made to `ENSURE_JTHREADOBJECT_NOT_VIRTUAL` (adding the `error` parameter).

Reply:
Updated.